test/dummy/db/migrate/20210425231534_create_spina_admin_journal_authors.spina_admin_journal_engine.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# This migration comes from spina_admin_journal_engine (originally 20201216211224)
+
+class CreateSpinaAdminJournalAuthors < ActiveRecord::Migration[6.0] # :nodoc:
+  def change
+    create_table :spina_admin_journal_authors, &:timestamps
```

Comment:
Nice!

Reply:
Cheers - this table will have an ORCID column in future I imagine. The way this is edited in the Spina interface needs work but that doesn't affect the frontend lol.